docs/core/tools/extensibility.md

```diff
@@ -145,7 +145,7 @@ defined beneath it.
 
 Consuming custom targets is done by providing a `<PackageReference>` that points to the package and its version inside the project that is being extended. Unlike the tools, the custom targets package does get included into the consuming project's dependency closure.
 
-Using the custom target depends solely on how you configure it. Since it's an MSBuild target, it can depend on a given target, run after another target and can also be manually invoked using the `dotnet msbuild /t:<target-name>` command.
+Using the custom target depends solely on how you configure it. Since it's an MSBuild target, it can depend on a given target, run after another target and can also be manually invoked using the `dotnet msbuild tt:<target-name>` command.
```

Comment:
-t instead of tt?

Reply:
funny, I caught an `msbuild -tt` but not the `msbuild  tt`